README.md

```diff
@@ -121,7 +121,14 @@ data into the new chain.
 However, any bump in the PATCH version should be compatible with existing histories
 (if not please open an [issue](https://github.com/tendermint/tendermint/issues)).
 
-For more information on upgrading, see [UPGRADING.md](./UPGRADING.md)
+For more information on upgrading, see [UPGRADING.md](./UPGRADING.md).
+
+### Supported Versions
+
+Because we are a small core team, we only ship patch updates, including security updates, 
+to the most recent minor release and the second-most recent minor release. Consequently, 
```

Comment:
While I agree with pushing people to upgrade, some of them can't. I think we still want to give them the most secure software possible. Also, back porting security patches does not take a lot of time usually, so maybe, we should reconsider our policy for security updates.

Reply:
I couldn't possibly agree more. It is acceptable to provide security updates-only as-is and on best effort basis, like many Free&Libre Open Source Software projects do. This would reassure downstreams, which then would feel confident to keep building their software on top of the Cosmos technology stack.

Reply:
That's great you are making explicit! We didn't know. I believe this statement is important for everyone. On our side, we are running version Tendermint 31 at IOV with Weave. We are planning to upgrade but we will need a few weeks, if you could provide this time the patch for the version 31, that would be great! I wanted also to congrats for the all work you do at Tendermint, you guys rocks 💪

Reply:
@antoineherzog I think @alessio is hooking you up on that one in this case :) 

@melekes I agree that there are definitely tradeoffs here, and I'm trying to document/articulate what the standard has been historically. I also don't think that the day of a security release is a good time to set new policy. I'll set some time aside on our next dev call to discuss and we can decide if we want to keep this same policy or change it up.